packages/vision/src/index.js

```diff
@@ -211,6 +212,19 @@ Vision.prototype.annotate = function(requests, callback) {
  * vision.detect(img, types, function(err, detection, apiResponse) {});
  *
  * //-
+ * // Run feature detection over a Buffer.
+ * //-
+ * var fs = require('fs');
+ *
+ * fs.readFile('image.jpg', function(err, contents) {
```

Comment:
I know this is exactly what we support, but examples sometimes appear as "the right way™ to do things". Is there another use case we could write up? @calibr -- how will you be using this feature?